doc/manual/src/installation/uninstall.md

```diff
@@ -69,7 +69,15 @@ which you may remove.
    # End Nix
    ```
 
-2. Stop and remove the Nix daemon services:
+   If you cannot edit these files directly, unlink them:
```

Comment:
In which case would it happen that one can't edit the files? And what happens if you unlink them? They won't be available for shell startup, right? That seems suspicious to me.